_posts/2016-06-24_mikemunsell_intro.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Introduction Post"
+date:   2016-06-24
+author: Mike Munsell
+---
+
+###Background
```

Comment:
To be on the safe side, I would put a space between the hashes and the words in your sub-titles